apps/src/templates/projects/ReportAbusePopUp.jsx

```diff
@@ -0,0 +1,271 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
+import RailsAuthenticityToken from '../../lib/util/RailsAuthenticityToken';
+import style from './project-card.module.scss';
+import Button from '@cdo/apps/templates/Button';
+import CheckBox from '@cdo/apps/componentLibrary/checkbox';
+import {connect} from 'react-redux';
+
+class UnconnectedReportAbusePopUp extends React.Component {
+  static propTypes = {
+    projectData: PropTypes.object,
+    recaptchaSiteKey: PropTypes.string,
+    onClose: PropTypes.func,
+    abuseUrl: PropTypes.string,
+    onReport: PropTypes.func,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showReportConfirmation: false,
+      captchaCompleted: false,
+      checkboxes: [
+        {name: i18n.abuseTypeCyberbullying(), checked: false},
+        {name: i18n.abuseTypeOffensiveHyphenated(), checked: false},
+        {name: i18n.abuseTypeInfringementHyphenated(), checked: false},
+        {name: i18n.abuseTypeOther(), checked: false},
+      ],
+      showRecaptcha: false,
+      submitButtonEnabled: false,
+      loadedCaptcha: false,
+    };
+    this.cancel = this.cancel.bind(this);
+    this.onCaptchaVerification = this.onCaptchaVerification.bind(this);
+    this.token = ''; // captcha token
+    this.onCaptchaExpiration = this.onCaptchaExpiration.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleCheckboxChange = this.handleCheckboxChange.bind(this);
+  }
+
+  componentDidUpdate() {
+    const {showRecaptcha, loadedCaptcha} = this.state;
+
```

Comment:
I think you may need to update conditions to prevent script loading more than once...maybe switch order to make it easier to read?
```
if (showRecaptcha && !loadedCaptcha) { // create
if (!showRecaptcha && loadedCaptcha) { // clean up

```